src/js/component.js

```diff
@@ -1443,7 +1443,9 @@ class Component {
     if (name === 'Player' && Component.components_[name]) {
       const Player = Component.components_[name];
 
-      if (Player.players && Object.keys(Player.players).length > 0) {
+      if (Player.players &&
+          Object.keys(Player.players).length > 0 &&
+          Object.keys(Player.players).map((playerName) => Player.players[playerName]).every(Boolean)) {
```

Comment:
Not sure whether we want a comment here but basically, if all the items in Player.players are null, you're allowed to register a custom player. This is partly so we can reset the Player component properly for the test.